architecture/workflows.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Workflows
+summary: Techniques for building stateful business workflows in distributed systems
+reviewed: 2023-07-18
+callsToAction: ['solution-architect']
+---
+
+Business processes typically involve various distributed components that need to be invoked in a coordinated manner. The Particular Service Platform supports both choreography and orchestration of workflows.
+
+Note: Choreography and orchestration are not mutually exclusive. The patterns may be combined at different levels of business workflows.
+
+## Choreography
+
+Choreographed workflows are implemented by an implicit flow of events between services instead of a central owner of a process. Services are highly decoupled by the use of [publish/subscribe](/architecture/messaging.md#communication-styles-publishsubscribe-pattern). Published messages are called _events_ because they describe a fact about something that happened to the rest of the system. Subscribers react to events as required. There is no central state of the workflow.
+
+![](/serviceinsight/images/overview-sequence-diagram.png)
+
+A choreographed event-driven workflow across multiple endpoints (visualized by a [ServiceInsight sequence diagram](/serviceinsight/sequence-diagram))
```

Comment:
I'm applying italics in an attempt to make it look more caption-y since Markdown doesn't really support that directly.
```suggestion
_A choreographed event-driven workflow across multiple endpoints, visualized by a [ServiceInsight sequence diagram](/serviceinsight/sequence-diagram)._
```